src/plugins/internal/InternalInstallPlugin.ml

```diff
@@ -165,14 +99,29 @@ let install pkg argv =
         (fun dn ->
            info (f_ "Creating directory '%s'") dn;
            BaseLog.register install_dir_ev dn)
-        tgt_dir;
+        (Filename.dirname tgt_file);
 
       (* Really install files *)
       info (f_ "Copying file '%s' to '%s'") src_file tgt_file;
       OASISFileUtil.cp ~ctxt:!BaseContext.default src_file tgt_file;
       BaseLog.register install_file_ev tgt_file
   in
 
+  (* Install the files for a library. *)
+  let install_lib_files findlib_name meta files =
+    let findlib_dir =
+      let s = Filename.concat (findlib_destdir ()) findlib_name in
+      fun () -> s
+    in
+    let files = ("", [meta]) :: files in
+    let f dir file =
+      let basename = Filename.basename file in
+      let tgt_fn = Filename.concat dir basename in
+      install_file ~with_destdir:false ~tgt_fn file findlib_dir
```

Comment:
If I change `~with_destdir` to `true`, all tests passes, but I think it's not correct.